src/services/config.ts

```diff
@@ -79,18 +77,18 @@ export class Config {
   }
   save() {
     const buf = Buffer.from(JSON.stringify(this.config, null, 2));
-    this.datastore.put(new Key('config'), buf);
+    void this.datastore.put(new Key('config'), buf);
```

Comment:
Why?

Reply:
I think making `save` an `async` function makes more sends, but if that means many code changes where save is used, then you can leave it as is for now.